src/main/java/seedu/address/model/task/Description.java

```diff
@@ -34,13 +34,27 @@ public Description(String description) {
         value = description;
     }
 
+    /**
+     * Constructs a default {@code Description}.
+     */
+    private Description() {
+        value = DEFAULT_DESCRIPTION;
+    }
+
     /**
      * Returns if a given string is a valid description.
      */
     public static boolean isValidDescription(String test) {
         return test.matches(VALIDATION_REGEX);
     }
 
+    /**
+     * Returns if a default Description with empty value;
```

Comment:
I think a better documentation is : construct an empty Description when user didn't provide the description field. Caveat: Only called when the user didn't key in this field.